go/plugins/dotprompt/dotprompt.go

```diff
@@ -66,6 +66,9 @@ type Prompt struct {
 	// The template for the prompt.
 	Template *raymond.Template
 
+	// The template for the prompt.
```

Comment:
This is the same comment as for the `Template` field.  Perhaps `// The parsed template.` and `// The unparsed template.`.

Reply:
Good catch! Done.